flink-python/pyflink/common/execution_config.py

```diff
@@ -673,7 +674,7 @@ def is_use_snapshot_compression(self):
         """
         return self._j_execution_config.isUseSnapshotCompression()
 
-    def set_use_snapshot_compression(self, use_snapshot_compression):
+    def set_use_snapshot_compression(self, use_snapshot_compression: bool):
         """
         Control whether the compression (snappy) for keyed state in full checkpoints and savepoints
```

Comment:
There are a few methods in ExecutionConfig returned nothing, e.g. enable_force_kryo, etc, what about making them return ExecutionConfig?

Reply:
The implementation of these methods is aligned with the java side, if the implementation is changed, it may not be convenient for users